README.md

```diff
@@ -271,3 +274,33 @@ ENTRYPOINT ["/go-binary"]
 ### Build final docker image
 
 `docker build -t docker-edgex-binary -f Dockerfile --build-arg BASE=base-build .`
+
+## Deployment On Sandbox
+
+The official site for Sandbox environment is **[https://jenkins.edgexfoundry.org/sandbox/](https://jenkins.edgexfoundry.org/sandbox/)**
+
+#### Get Access To Sandbox
+
+The official process to request for access to sandbox is provided in the link **[Here](https://https://lf-releng-docs.readthedocs.io/en/latest/jenkins-sandbox.html#get-access-to-the-jenkins-sandbox)**
```

Comment:
The link URL is broken due to having "https://https://" at the beginning.

Reply:
fixed